src/Aspire.Hosting.Garnet/GarnetBuilderExtensions.cs

```diff
@@ -0,0 +1,131 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire.Hosting.ApplicationModel;
+using Aspire.Hosting.Garnet;
+using Aspire.Hosting.Utils;
+
+namespace Aspire.Hosting;
+
+/// <summary>
+/// Provides extension methods for adding Garnet resources to the application model.
+/// </summary>
+public static class GarnetBuilderExtensions
+{
+    /// <summary>
+    /// Adds a Garnet container to the application model.
+    /// </summary>
+    /// <example>
+    /// <remarks>Use in AspireHost</remarks>
+    /// <code>
+    /// var builder = DistributedApplication.CreateBuilder(args);
+    ///
+    /// var api = builder.AddProject&lt;Projects.Api&gt;("api") 
```

Comment:
I would change this to match how we show in samples:

```csharp
var garnet = builder.AddGarnet("garnet");

var api = builder.AddProject<Projects.Api>("api)
    .WithReference(garnet);
```

Reply:
done